storybookjs/storybook
# Bug: disable cache in dev-mode

Closes #21443

## What I did

I just added a no-cache control header on every request coming from the storybook dev-server

## How to test

1. Run a sandbox for template, e.g. `yarn task --task sandbox --start-from auto --template react-vite/default-ts`
2. Open Storybook in your browser
3. check the request in the network tab, they should all show no-cache headers
